docs/assets/fragments/connectivity.txt

```diff
@@ -0,0 +1,45 @@
+1. You will need the login and password for the admin user to access the
+    cluster. Use `kubectl get secrets` command to see the list of Secrets
```

Comment:
I assume we do not need this file because it's related to the mongo db connectivity. We can get the same from the GKE guide

Reply:
Removed everything following the  **Create the EKS cluster** section

Reply:
@catalinaadam you missed deleting the `docs/assets/fragments/connectivity.txt` file